src/ComponentFolders/LoginPage/Login.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import Image from "./Images/LoginSofa.svg"
```

Comment:
The image file has been removed (Probably by mistake) but it doesn't exist in the file.

Reply:
I will check again

Reply:
I just remane the image file becasue the name had () and it causes an error while importing it

Reply:
Instead of Login(Sofa) it is now LoginSofa

Reply:
I think you must've forgotten to add it, it doesn't exist in the file, or maybe you pushed and then added the image 

![image](https://user-images.githubusercontent.com/110013797/209652608-07600e64-3d11-46b5-a4ea-1995546e7bc3.png)